File: Programmers/완주하지 못한 선수.cpp
#include <string>
#include <vector>
#include <iostream>
#include <algorithm>
using namespace std;

string solution(vector<string> p, vector<string> c) {
	string answer = "";
	sort(p.begin(), p.end());
	sort(c.begin(), c.end());
	for (int i = 0; i < c.size(); i++) {
		if (p[i] != c[i])
			return p[i];
	}

	return p[p.size() - 1];
}

int main() {
	vector<string> participant = { "leo", "kiki", "eden" },
		completion = { "eden","kiki" };
	cout << solution(participant, completion);
	system("pause");
}

File: Programmers/완주하지 못한 선수.java
import java.util.Arrays;

class Solution {
    public String solution(String[] p, String[] c) {
        Arrays.sort(p);
        Arrays.sort(c);
        for(int i = 0;i < c.length; i++)
            if(!p[i].equals(c[i]))
                return p[i];
        return p[p.length-1];
    }
}